README.md

```diff
@@ -61,7 +63,7 @@ Many of these features require the [AWS SAM CLI](https://github.com/awslabs/aws-
 Serverless Application Model ([SAM](https://aws.amazon.com/serverless/sam/)) website for more information on 
 installation of the SAM CLI.
 
-**SAM features are currently limited to Java and Python only**
+**SAM features are currently limited to Java, Python, Node.js, and .NET Core only**
```

Comment:
Nit: Our list now covers most of the SAM cli supported languages, we should probably remove "only" and "currently limited"